File: problemas/iniciante/2987/2987.ts
process.stdin.resume();
process.stdin.setEncoding('utf8');

let input: string = "";

process.stdin.on("data", (chunk: string) => {
    input += chunk;
});

process.stdin.on("end", () => {
    let lines: string[] = input.split('\n');
    let L: string = lines.shift()!
    let alfabeto: string[] = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z'];

    for (let letra of alfabeto) {
        if (letra.toUpperCase() === L) {
            console.log(alfabeto.indexOf(letra) + 1);
        }
    }

});

File: problemas/iniciante/2987/2987.py
alfabeto = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 
            'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 
            'u', 'v', 'w', 'x', 'y', 'z']

L = input()
for letra in alfabeto:
    if letra.upper() == L:
        print(alfabeto.index(letra) + 1)
        break
